tests/Integration/Framework/Test_ProxyDB.py

```diff
@@ -0,0 +1,555 @@
+""" This is a test of the ProxyDB
+    It supposes that the DB is present and installed in DIRAC
+"""
```

Comment:
Please add this to https://github.com/DIRACGrid/DIRAC/blob/integration/tests/Integration/all_integration_server_tests.sh